src/platform/i386/lapic.c

```diff
@@ -95,7 +116,7 @@ u32_t               lapic_timer_calib_init   = 0;
 static void
 lapic_write_reg(u32_t off, u32_t val)
 {
-	*(u32_t *)(lapic + off) = val;
+	*(u32_t volatile *)(lapic + off) = val;
```

Comment:
again a nitpick: I don't remember if CSG says anything about the placement of `volatile` keyword. But I think in most places we use like `volatile u32_t *`. 

Thanks for fixing this! I don't know how it was working fine for timer configurations even without. 
Perhaps this will fix a few issues where the timer interrupts were not triggered in `Qemu`. Hope it does!

Reply:
indeed, storage modifiers before data types.

Reply:
dito.